TicTacToe/Logic/PlayerEventArgs.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System;
+
+namespace TicTacToe.Logic
+{
+    public class PlayerEventArgs : EventArgs
```

Comment:
Naming: The class name isn't appropriate, due to the property "AttemptsCount" - what does it mean in the context of a player? The class name should reflect the name of the event for which we have both a player and attempts count